collect_app/src/androidTest/java/org/odk/collect/android/regression/EspressoTestUtilities.java

```diff
@@ -0,0 +1,162 @@
+package org.odk.collect.android.regression;
```

Comment:
I think this class should be one level up. You want to have all regression cases in one package what's ok but we will add more tests (out of this package) which could use the same helpers.